bookkeeper-http/http-server/src/main/java/org/apache/bookkeeper/http/HttpRouter.java

```diff
@@ -51,6 +51,8 @@
     public static final String BOOKIE_STATE_READONLY        = "/api/v1/bookie/state/readonly";
     public static final String BOOKIE_IS_READY              = "/api/v1/bookie/is_ready";
     public static final String BOOKIE_INFO                  = "/api/v1/bookie/info";
+    public static final String SUSPEND_COMPACTION           = "/api/v1/bookie/compaction/suspend";
+    public static final String RESUME_COMPACTION            = "/api/v1/bookie/compaction/resume";
```

Comment:
I prefer to use `gc` endpoint instead of `compaction`, It will introduce more confusion.